views/partials/foot.jade

```diff
@@ -2,9 +2,9 @@ if scripts && scripts.length
 	each script in scripts
 		if !script.inHead
 			if script.content
-					script(async=script.async crossorigin=script.crossorigin defer=script.defer integrity=script.integrity src=script.src text=script.text type=script.type)!= script.content
+					script(async=script.async crossorigin=script.crossorigin defer=script.defer integrity=script.integrity nomodule=script.nomodule nonce=script.nonce referrerpolicy=script.referrerpolicy src=script.src text=script.text type=script.type)&attributes(script.dataAttributes || {})!= script.content
```

Comment:
I would prefer the `script.dataAttributes || {}` happen in JavaScript, not Jade. This will make it easier to move away from Jade.